spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewritePositionDeleteFilesAction.java

```diff
@@ -192,7 +192,7 @@ public void testUnpartitioned() throws Exception {
             .option(SizeBasedFileRewriter.REWRITE_ALL, "true")
             .execute();
     List<DeleteFile> newDeleteFiles = deleteFiles(table);
-    assertThat(newDeleteFiles).as("Expected 1 new delete file").hasSize(1);
+    assertThat(newDeleteFiles).as("Expected new delete file(s)").hasSize(1);
```

Comment:
Can we remove language like 'expected'?  I think assertThat already puts that?

Reply:
Removed.